### DISCUSSION TITLE
disable uppercase the first character for ModelLabel

### DISCUSSION THREAD

**Discussion by ihahachi** [filamentphp/filament#6112] [Help] [answered]
How to disable uppercase the first character of title ?

```
public static function getModelLabel(): string
  {
      return __('espece.mode_preparation');
  }
```
![image](https://user-images.githubusercontent.com/7762172/229659960-e34d8d32-594e-4de8-a1fc-eba7f3d03b4c.png)

**Comment by danharrin** [comment, marked as answer]
Set the `getNavigationLabel()` explicitly

**Reply by ihahachi** [reply]
it's working thanks Dan
